dts/bindings/spi/nordic,nrf-spis.yaml

```diff
@@ -14,3 +14,11 @@ properties:
     description: |
         Default character. Character clocked out when the slave was not
         provided with buffers and is ignoring the transaction.
+
+  csn-pin-anomaly-109-workaround:
+    type: int
+    description: |
+      Enabling the nRF52832 errata 109 workaround requires the CSN pin to be
+      known by the driver, even when PINCTRL is enabled. There is no way to
+      extract the SPIS_CSN knowledge from the pinctrl node at compile time, so
+      we are forced to specify the value a second time here.
```

Comment:
This needs be fixed in the nrfx_spis driver. Then this property will not be needed.
Let me provide a PR to hal_nordic that will address this.

Reply:
@JordanYates Here's the promised PR: https://github.com/zephyrproject-rtos/hal_nordic/pull/135.

Reply:
@JordanYates can you please point `west.yml` in this PR to the hal_nordic PR above?

Reply:
done

Reply:
@JordanYates we added this change to the recent nrfx release. Please change revision in manifest to https://github.com/zephyrproject-rtos/hal_nordic/pull/137